### PR TITLE
Add swalot to submitfilter (HPC-5683)

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: lib/vsc/jobs/pbs/clusterdata.py [hunk 1]
```diff
@@ -88,6 +88,13 @@
         'NP': 2,
         'NP_LCD': 2,
     },
+    'swalot': {
+        'PHYSMEM': 131791292 << 10,
```
Comment (itkovian) [inline lib/vsc/jobs/pbs/clusterdata.py]: Weren't we going to use GiB? :p

Comment (stdweird) [inline lib/vsc/jobs/pbs/clusterdata.py]: ?

File: test/testpbs/pbs_get_jobs_data_master21_t51 [hunk 2]
```diff
@@ -0,0 +1 @@
...ob'], 'Priority': ['0'], 'ctime': ['1454170105'], 'Rerunable': ['True'], 'server': ['master21.swalot.gent.vsc'], 'queue': ['short'], 'queue_type': ['E'], 'Output_Path': ['gligar01.gligar.gent.vsc:/vscmnt/gent_vulpix/_/user/data/gent/vsc400/vsc40075/JUBE-benchmarks/JUBE_jobs_output/swalot/$PBS_JOBID.out'], 'Job_Owner': ['vsc40075@gligar01.gligar.gent.vsc'], 'Job_Name': ['JUBE-stream-1b_openmp'], 'Variable_List': {'PBS_O_LOGNAME': ['vsc40075'], 'PBS_O_SHELL': ['/bin/bash'], 'PBS_O_HOME': ['/user/home/gent/vsc400/vsc40075'], 'PBS_O_HOST': ['gligar01.gligar.gent.vsc'], 'PBS_O_QUEUE': ['default'], 'PBS_O_WORKDIR': ['/vscmnt/gent_vulpix/_/user/data/gent/vsc400/vsc40075/JUBE-benchmarks'], 'PBS_O_LANG': ['en_US.UTF-8'], 'PBS_O_PATH': ['/user/data/gent/vsc400/vsc40075/JUBE/bin:/user/home/gent/vsc400/vsc40075/.cabal/bin:/user/home/gent/vsc400/vsc40075/bin:/user/data/gent/vsc400/vsc40075/JUBE/bin:/user/home/gent/vsc400/vsc40075/.cabal/bin:/user/home/gent/vsc400/vsc40075/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/opt/ibutils/bin'], 'PBS_O_SERVER': ['master21.swalot.gent.vsc'], 'PBS_O_SUBMIT_FILTER': ['/usr/bin/submitfilter'], 'PBS_O_MAIL': ['/var/spool/mail/vsc40075']}}}
```
Comment (itkovian) [inline test/testpbs/pbs_get_jobs_data_master21_t51]: Does this have to be a single line? It makes it hard to read and to see what is there. Also, does this contain data for the named nodes extra thingie to be tested?

File: lib/vsc/jobs/pbs/clusterdata.py (hunk 1 above)
Comment (itkovian) [inline lib/vsc/jobs/pbs/clusterdata.py]: 128 << 30 ?

File: test/testpbs/pbs_get_jobs_data_master21_t51 (hunk 2 above)
Comment (stdweird) [inline test/testpbs/pbs_get_jobs_data_master21_t51]: i'll check (it's generated) and yes (this data actually trigger the issue i'm trying to fix)

File: lib/vsc/jobs/pbs/clusterdata.py (hunk 1 above)
Comment (stdweird) [inline lib/vsc/jobs/pbs/clusterdata.py]: but that's whole point. it's not `128 << 30`, what pbsnodes reports is what is used/seen by pbs_server.

**Comment by stdweird** [comment]
@itkovian now with pretty data

File: lib/vsc/jobs/pbs/clusterdata.py (hunk 1 above)
Comment (itkovian) [inline lib/vsc/jobs/pbs/clusterdata.py]: OK, 125 << 30 then :) But yes, its fine as is. It's just hard to read.

**Pull request merged**